src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/AuthenticatedClient.kt

```diff
@@ -103,6 +104,12 @@ class AuthenticatedClient(
   fun getUserDetails(userId: UserId) =
     getRequest("/users/${userId.value}", UserDetailsResponse::class.java)
 
+  fun localDeliveryUnit(): List<LocalDeliveryUnitResponse> =
+    sendGetRequest("/reference-data/local-delivery-unit")
```

Comment:
should be plural? /local-delivery-units

Reply:
I thought that, then thought we were doing singular...1 second...

Reply:
👍  I won't approve the rest, will leave that to someone who knows Kotlin